go.mod

```diff
@@ -41,11 +41,11 @@ require (
 	github.com/mitchellh/hashstructure v1.1.0
 	github.com/mitchellh/mapstructure v1.5.1-0.20231216201459-8508981c8b6c
 	github.com/oklog/ulid v1.3.1
-	github.com/open-telemetry/opentelemetry-collector-contrib/exporter/fileexporter v0.98.0
```

Comment:
Could we add a couple of newlines around this block of OTel dependencies or add a separate `require` block for them to indicate that they are all related?  

This would then give us an opportunity to add a comment here explaining why we're pinning to `0.97.0` for now and what is the criteria for undoing that pinning.  That way, when dependabot tries to automatically bump the version, whoever reviews those PRs would immediately know whether to approve them or not.

Reply:
i think we could separate